ui/src/reusable_ui/components/page_layout/test/PageHeader.test.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import {shallow} from 'enzyme'
+
+import {Page} from 'src/reusable_ui'
+
+describe('PageHeader', () => {
+  let wrapper
+
+  const wrapperSetup = (
+    overrideProps = {},
+    overrideLeft = (
+      <Page.Header.Left>
+        <Page.Title title="Yeehaw" />
+      </Page.Header.Left>
+    ),
+    overrideCenter = (
+      <Page.Header.Center>
+        <p>Center!</p>
+      </Page.Header.Center>
+    ),
+    overrideRight = <Page.Header.Right />
+  ) => {
+    const props = {
+      fullWidth: false,
+      inPresentationMode: false,
+      ...overrideProps,
+    }
+
+    return shallow(
+      <Page.Header {...props}>
+        {overrideLeft}
+        {overrideCenter}
+        {overrideRight}
+      </Page.Header>
+    )
+  }
+
+  beforeEach(() => {
+    jest.resetAllMocks()
+    wrapper = wrapperSetup()
```

Comment:
might make sense to omit the `beforeEach` block altogether, since in half the tests you overwrite `wrapper` again anyway. I think the tests might read easier this way, since they would be more self-contained.